blocks/comparison/comparison.css

```diff
@@ -0,0 +1,108 @@
+.comparison {
+  font-size: 14px;
+  max-width: var(--grid-container-width);
+  margin: 0 auto 40px;
+  padding: 20px;
+  background: var(--color-gray-100);
+}
+
+.comparison .comparison-wrapper {
+  max-height: 80vh;
+  overflow: auto;
+}
+
+.comparison table {
+  width: 100%;
+  border-spacing: 0;
+  overflow: scroll;
+  min-width: 600px;
```

Comment:
Having a min-width seems a bit risky. What it there are only a couple of columns and the content would naturally fit in a smaller width? It seems like the size of the table should come from the size of the cells.

Reply:
Yes, probably trying to solve the wrong thing here, removed.

Reply:
The issue is without a min-width text heavy columns are stretched vertically on mobile, Example 2, but as you noticed small tables had unnecessary scrolling, Example 1. Going to prioritize having more content on screen so keeping this removed.